src/main/java/com/adyen/model/checkout/RecurringDetail.java

```diff
@@ -0,0 +1,263 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ *
+ * Adyen Java API Library
+ *
+ * Copyright (c) 2018 Adyen B.V.
+ * This file is open source and available under the MIT license.
+ * See the LICENSE file for more info.
+ */
+
+
+package com.adyen.model.checkout;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * RecurringDetail
+ */
+public class RecurringDetail {
```

Comment:
where do we use that class?

Reply:
This class is an auto generated model used in oneClickPaymentMethods mthod of PaymentMethodsResponse class. Please see specification below. I believe oneClickPaymentMethods should be called by calling APIs (client APIs receiving the PaymentMethodsResponse ).
PaymentMethodsResponse:
      properties:
        oneClickPaymentMethods:
          description: Detailed list of one-click payment methods.
          items:
            $ref: '#/components/schemas/RecurringDetail'
          type: array